src/coreclr/jit/emitwasm.h

```diff
@@ -1,6 +1,8 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+typedef uint32_t code_t;
```

Comment:
Unless this is useful for common code, I don't think it is useful to have a `code_t` like this. WASM instructions can have very long encodings when multiple relocs get involved (or vector constants).